pkg/alertmanager/multitenant.go

```diff
@@ -22,23 +23,28 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/cortexproject/cortex/pkg/alertmanager/alerts"
+	"github.com/cortexproject/cortex/pkg/ring"
+	"github.com/cortexproject/cortex/pkg/ring/kv"
 	"github.com/cortexproject/cortex/pkg/tenant"
 	"github.com/cortexproject/cortex/pkg/util"
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 	"github.com/cortexproject/cortex/pkg/util/services"
 )
 
-var backoffConfig = util.BackoffConfig{
-	// Backoff for loading initial configuration set.
-	MinBackoff: 100 * time.Millisecond,
-	MaxBackoff: 2 * time.Second,
-}
-
 const (
 	// If a config sets the webhook URL to this, it will be rewritten to
 	// a URL derived from Config.AutoWebhookRoot
 	autoWebhookURL = "http://internal.monitor"
 
+	// Reasons for (re)syncing alertmanager configurations from object storage.
+	reasonPeriodic   = "initial"
+	reasonInitial    = "periodic"
```

Comment:
Values are inverted:

```suggestion
	reasonPeriodic   = "periodic"
	reasonInitial    = "initial"
```